test/grpc/test_nexus.js

```diff
@@ -836,6 +846,25 @@ describe('nexus', function () {
       common.jsonrpcCommand('/tmp/target.sock', 'nvmf_subsystem_remove_ns', args, done);
     });
 
+    it('should still have bdev of removed child after remove event', (done) => {
+      common.jsonrpcCommand(null, 'bdev_get_bdevs', (err, out) => {
```

Comment:
Should we also check that nexus child has also changed its state after removing the underlaying namespace ?

Reply:
This is sort of covered by the cargo test replica_stop_cont but that's actually more a data plane test. I guess it won't hurt to add another test here.